### ISSUE TITLE
Can't join Teams: /channels/link never shows a link

### ISSUE THREAD

**Issue by EliahKagan** [Charcoal-SE/metasmoke#707]
While logged in to metasmoke, I attempted to join the Teams group by pasting the email address from metasmoke.erwaysoftware.com/channels/email at stackoverflow.com/c/charcoal/join/submit. Everything appeared to work but no link ever appeared at https://metasmoke.erwaysoftware.com/channels/link. It continues to say:

> We haven't received the email yet. Check back in a few minutes.

I tried this a few times, waiting considerable time in between in case the problem was simply that I hadn't waited long enough. Each time, even after many (at least once, over ten) hours, there was no change. I didn't notice any other indication of anything going wrong.

All attempts have been in the last few days, and thus after metasmoke was moved to the new server. I don't actually know if that's related to the problem or not, though, since I've never attempted to join Teams before this.

**Comment by Undo1** [comment]
Sorry for the long response time, been busy RL.

Norepro today. Are you reloading the page? If you can try again and give screenshots + exact process, it'd be useful.

**Comment by EliahKagan** [comment]
Thanks--I've tried again today and was also unable to reproduce the problem. It succeeded.

Reloading the page didn't cause the link to come up at https://metasmoke.erwaysoftware.com/channels/link, but repeating the process from the beginning did work.

I'm not sure why it worked this time. I had repeated it before, though not in the past few days. I don't *think* I did anything differently this time, in terms of what steps I followed and in what order. I did use a different machine and browser this time, though I don't expect that was significant. I can't be certain I hadn't made some mistake before.

I can give the details and screenshots of what I did this time, and will do so if requested, but I'm guessing that's not really valuable, since it's the time it *didn't* work that's of interest. Since I no longer have the problem and I don't know of any way to reproduce it, I'm closing this as not reproducible.
